clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/AutoElasticsearchQueryMapper.scala

```diff
@@ -92,6 +98,14 @@ class AutoElasticsearchQueryMapper[
           esName.stripSuffix("_end"),
           value.asInstanceOf[OffsetDateTime]
         )
+
+      case tpe
+          if indexVersion > 1 &&
```

Comment:
The way that `1` is used here seems like a magic number, e.g. it has a special meaning to the code. Replacing it with a name could make the code clearer, and will help a future maintainer if they want to find out everywhere this special version number is used.

Reply:
Another option would be to replace `indexVersion` with an object that computes `queryVal()` so the object encapsulates this logic, e.g. add it to `ElasticSearchIndex` instead of adding the version number.